docs/workforce-management-framework/topics/con-KeycloakVsPassport.adoc

```diff
@@ -0,0 +1,37 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} currently supports two security implementations
+Keycloak and Passport.js.
+
+== Passport.js
+Passport.js is a lightweight authentication middleware for Node.js that is Extremely flexible and modular.
+A comprehensive set of strategies support authentication using a username and password, Facebook, Twitter, and more.
+For more information see link:https://passport.org[passport.org].
+
+== Keycloak
+Keycloak is a full identity management solution with many features
+
+- link:https://keycloak.gitbooks.io/documentation/server_admin/topics/identity-broker.html[Identity brokering]
```

Comment:
I think that we add the features to a list with no url links and at the bottom just have For a full feature list, see link:https://keycloak.gitbooks.io/documentation/server_admin/topics/overview/features.html[Keycloak].

Why? Reducing possible broken links